### PR TITLE
Bump Citus PGXN to 9.3.4

### PR DESCRIPTION
**Pull request by onurctirtir**

### CODE REVIEW

File: META.json [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
    "abstract": "Scalable PostgreSQL for real-time workloads",
    "description": "Citus horizontally scales PostgreSQL across commodity servers using sharding and replication. Its query engine parallelizes incoming SQL queries across these servers to enable real-time responses on large datasets.",
-   "version": "9.3.0",
+   "version": "9.3.4",
    "maintainer": "\"Citus Data\" <packaging@citusdata.com>",
    "license": "agpl_3",
    "provides": {
       "citus": {
          "abstract": "Citus Distributed Database",
          "file": "README.md",
```
Comment (onurctirtir) [inline META.json]:
```suggestion
         "file": "src/backend/distributed/sql/citus--9.3-2--9.4-1.sql"
```

File: META.json [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
    "description": "Citus horizontally scales PostgreSQL across commodity servers using sharding and replication. Its query engine parallelizes incoming SQL queries across these servers to enable real-time responses on large datasets.",
-   "version": "9.3.0",
+   "version": "9.3.4",
    "maintainer": "\"Citus Data\" <packaging@citusdata.com>",
    "license": "agpl_3",
    "provides": {
       "citus": {
          "abstract": "Citus Distributed Database",
-         "file": "README.md",
+         "file": "src/backend/distributed/sql/citus--9.3-2--9.4-1.sql"
```
Comment (onurctirtir) [inline META.json]:
```suggestion
         "file": "src/backend/distributed/sql/citus--9.3-2--9.4-1.sql",
```

File: META.json (hunk 1 above)
Comment (onurctirtir) [reply META.json]: lets use migration script, or should I put citus.so ? @hanefi

File: META.json (hunk 2 above)
Comment (onurctirtir) [reply META.json]: missing comma

File: META.json (hunk 1 above)
Comment (hanefi) [reply META.json]: `src/backend/distributed/sql/citus--9.3-2--9.4-1.sql` should not be present in this package.

I think it's better to keep maintanence costs low. Let's have a dummy value here that does not need to be updated across versions. 

I wonder why we have `RREADME.md` here. How does `citus.so` sound?

Comment (onurctirtir) [reply META.json]: sounds good, the only thing I'm not sure is where the `citus.so` is placed in directory hierarchy, would directly writing ``citus.so` work ?

Comment (hanefi) [reply META.json]: In the past, we made a typo and the package still got created. So I don't think there is any checks for validity of file paths here

Comment (onurctirtir) [reply META.json]: done

**Review by hanefi — APPROVED** [review]
:shipit:

**Pull request merged**